### DISCUSSION TITLE
LaTeX 字体和行距问题

### DISCUSSION THREAD

**Discussion by HT0258** [sikouhjw/gdutthesis#32] [Q&A] [answered]
有两个问题想请教下：
1. word模板中三级标题字号与正文一致，Latex中好像不一致，并且字体应该都是宋体；
2. 公式与段落之间靠得非常近，是1.5倍行距吗，显得非常拥挤。

**Comment by sikouhjw** [comment, marked as answer]
# 第一个问题
请阅读《撰写规范》[^1]的第三部分：打印要求。
当《撰写规范》与《广东工业大学研究生学位论文撰写规范格式模板》[^2]矛盾时，**一般**以撰写规范为准。

# 第二个问题
1. word 对公式前后间距的处理逻辑与 latex 不同
2. 关于 word 行距的讨论：https://github.com/sikouhjw/gdutthesis/issues/6
3. 关于 latex 数学公式与正文的间距：https://github.com/sikouhjw/gdutthesis/issues/12
4. 看完上面，结论是：word 模板及规范对公式相关的内容没有任何参考价值，请不要以 word 的思维方式来判断 latex 模板是否符合标准，因为 word 模板本身就没有一个严格的标准

[^1]: https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073
[^2]: https://yjs.gdut.edu.cn/guangdonggongyedaxueyanjiushengxueweilunwenzhuanxieguifangeshimoban.docx

**Comment by sikouhjw** [comment]
> 当《撰写规范》与《广东工业大学研究生学位论文撰写规范格式模板》矛盾时，**一般**以撰写规范为准。

更何况这里并没有矛盾，你说的
> 三级标题字号与正文一致，Latex中好像不一致，并且字体应该都是宋体；

是师兄师姐给的『祖传』模板吧

**Comment by HT0258** [comment]
> # 第一个问题
> [撰写规范](https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073)第三部分 打印要求，请看。 当[撰写规范](https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073)与[广东工业大学研究生学位论文撰写规范格式模板](https://yjs.gdut.edu.cn/guangdonggongyedaxueyanjiushengxueweilunwenzhuanxieguifangeshimoban.docx)矛盾时，一般以撰写规范为准。
> 
> # 第二个问题
> 1. word 对公式前后间距的处理逻辑与 latex 不同
> 2. 关于 word 行距的讨论：[模板行距问题 #6](https://github.com/sikouhjw/gdutthesis/issues/6)
> 3. 关于 latex 数学公式与正文的间距：[正文与数学公式之间的间距小怎么办？ #12](https://github.com/sikouhjw/gdutthesis/issues/12)
> 4. 看完上面，结论是：word 模板及规范对公式相关的内容没有任何参考价值，请不要以 word 的思维方式来判断 latex 模板是否符合标准，因为 word 模板本身就没有一个严格的标准

**Comment by HT0258** [comment]
好的，已知晓，谢谢
